parsers/ES_IB.py

```diff
@@ -1,18 +1,13 @@
-#!/usr/bin/env python3
 
 import logging
+
 from arrow import get
+from ree import BalearicIslands, Formentera, Ibiza, Mallorca, Menorca
 from requests import Session
-from ree import (Formentera, Ibiza,
-                 Mallorca, Menorca,
-                 BalearicIslands)
-# package "ree" is used to parse data from www.ree.es // maintained on github by @hectorespert
 
 from .lib.exceptions import ParserException
 from .lib.validation import validate, validate_production_diffs
 
-## Guess we'll need to figure these out later?! Adapted from ES-CN:
```

Comment:
This comment should stay, minimum zone demands are specified to be 0, that is clearly a placeholder. Maybe add a `TODO:` to make it clearer? Otherwise, remove the use of FLOORS in this script altogether, it's optional in `validate`.